docs/jwst/tweakreg/README.rst

```diff
@@ -162,17 +162,17 @@ telescope pointing will be identical in all these images and it is assumed
 that the relative positions of (e.g., NIRCam) detectors do not change.
 Identification of images that belong to the same "exposure" and therefore
 can be grouped together is based on several attributes described in
-`~jwst.datamodels.ModelLibrary`. This grouping is performed automatically
+`~jwst.datamodels.library.ModelLibrary`. This grouping is performed automatically
 in the ``tweakreg`` step using the
-`~jwst.datamodels.ModelLibrary.group_names` property.
+`~jwst.datamodels.library.ModelLibrary.group_names` property.
```

Comment:
This is more of a docstring improvement, but I think it would be better for this to say
"This grouping is performed automatically in the ``tweakreg`` step. The groups are accessible through the `group_names` attribute of the library.